### ISSUE TITLE
Bug: streamdeck-ui crashes on start when no Stream Deck is plugged in

### ISSUE THREAD

**Issue by TuringTux** [timothycrosley/streamdeck-ui#77]
When no Stream Deck is plugged in, streamdeck-ui crashes on start:

```
Traceback (most recent call last):
  File "/usr/bin/streamdeck", line 33, in <module>
  File "/usr/lib/python3.8/site-packages/streamdeck_ui/gui.py", line 269, in start
  File "/usr/lib/python3.8/site-packages/streamdeck_ui/gui.py", line 199, in build_device
  File "/usr/lib/python3.8/site-packages/streamdeck_ui/gui.py", line 129, in build_buttons
  File "/usr/lib/python3.8/site-packages/streamdeck_ui/api.py", line 133, in get_deck
KeyError: None
```

In my opinion, it would be nice to catch this exception and show a popup (or at least a message on the terminal). I installed the software a few days before I received the Stream Deck to check whether it starts on my computer or not and was a little scared to see a fresh installation crash (especially because I did not have the time to misconfigure it before).

I could try to implement this, but I'd then have to take some time to work through the code.

Edit: For any future visitors with the same problem: When the Stream Deck is plugged in, everything works just fine for me.

**Comment by TuringTux** [comment]
This is probably fixed in #26

**Comment by dodgyrabbit** [comment]
This has been fixed.
